faq/audio.rst

```diff
@@ -5,10 +5,12 @@ noises, try to increase the timeout ::
 
     SOUND_POWER_SAVE_ON_BAT=10
 
-or disable power save completely
+or disable power save completely ::
 
     SOUND_POWER_SAVE_ON_BAT=0
 
-.. seealso::
+see also ::
```

Comment:
Please remove your change to lines 12-14(16) from the PR. Reason: this would change the "see also" block to a code block, which doesn't fit here. I'd like to keep the block style.